### ISSUE TITLE
[Bug] [Share] Context menu doesn't show new share sheet

### ISSUE THREAD

**Issue by jonalmeida** [mozilla-mobile/fenix#2933]
## Steps to reproduce
1. Click and hold a link on web content.
1. Click Share.

### Expected behavior
* To see the new share sheet. (right of screenshot)

### Actual behavior
* The old share sheet. (left of screenshot)

~~This may require a~~c work - needs investigation.- Needs a-c work.

![](https://user-images.githubusercontent.com/1370580/58499325-e7077000-814d-11e9-8674-80e23b0ffce8.png)

**Comment by sblatz** [comment]
p3 but blocked on a decision about keeping the custom sheet or not.

**Comment by jonalmeida** [comment]
Removing the sizing estimation since it was meant for a-c and I've created a separate issue there instead.
